### PR TITLE
copyedit prompt-design vignette: intro & Logistics

### PR DESCRIPTION
**Pull request by lindbrook**

### CODE REVIEW

**Comment by lindbrook** [comment]
I'm not sure about the "Logistics". Unless it's a technical term in the literature, it's more a set of (opinionated/strong) recommendations, suggestions or best practices.

File: vignettes/prompt-design.Rmd [hunk 1]
```diff
@@ -23,41 +23,42 @@ chat_claude <- function(...) {
...
 
-As you iterate on the prompt, it's a good idea to build up a small set of challenging examples that you can regularly re-check with your latest version of the prompt. Currently you'll need to do this by hand, but we hope to eventually also provide tools that help you do this a little more formally.
+As you iterate the prompt, it's a good idea to build up a small set of challenging examples that you can regularly re-check with your latest version of the prompt. Currently you'll need to do this by hand, but we hope to eventually provide tools that'll help you do this a little more formally.
 
-Unfortunately, however, you won't see these logistics in action in this vignette, since we're keeping the prompts short and inline to make it easier for you to grok what's going on.
+Unfortunately, you won't see these logistics in action in this vignette since we're keeping the prompts short and inline to make it easier for you to grok what's going on.
```
Comment (lindbrook) [inline vignettes/prompt-design.Rmd:61]: inline or online?

Comment (hadley) [reply vignettes/prompt-design.Rmd:61]: Inline

**Comment by hadley** [comment]
Best practices might be better than logistics.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**